src/commands/maps/remove_m_from_pool.ts

```diff
@@ -0,0 +1,42 @@
+import {ApplicationCommandOptionTypes} from "discord.js/typings/enums";
```

Comment:
rename to `remove_map` as explained above

Reply:
Resloved.